src/x/opentracing/tracing.go

```diff
@@ -21,9 +21,15 @@
 package opentracing
 
 import (
+	"context"
 	"fmt"
 	"io"
+	"runtime"
+	"strings"
+	"time"
 
+	lightstep "github.com/lightstep/lightstep-tracer-go"
+	"github.com/m3db/m3/src/x/instrument"
```

Comment:
nit: `github.com/m3db/m3...` imports in the middle separated from the third party exports at the bottom? (linting seems to be turned off right now)